app/controllers/v1/submission_files_controller.rb

```diff
@@ -1,6 +1,8 @@
 class V1::SubmissionFilesController < APIController
   deserializable_resource :submission_file, only: %i[update]
 
+  skip_before_action :reject_without_user!, except: %i[show]
```

Comment:
Again, don't think so, but unsure.

Reply:
confirmed that this is being skipped because we're currently using this endpoint for the `local_injest.js` script. I think we should probably add a comment in the code explaining why we are skipping this.